### ISSUE TITLE
US holidays wrong dates for some years

### ISSUE THREAD

**Issue by a-nasstrom** [nager/Nager.Date#674]
### Endpoint

https://date.nager.at/api/v2/publicholidays/2023/US

### More information

2022/US, 2023/US, 2028/US +
for some years, the date is shifted by 1-2 days
<img width="205" alt="image" src="https://github.com/user-attachments/assets/20a9a93a-2e3b-4211-a86a-22a003ef7912">
<img width="179" alt="image" src="https://github.com/user-attachments/assets/645263a5-108f-4194-bf6d-2275b1cb971c">

<img width="380" alt="image" src="https://github.com/user-attachments/assets/6493b0ac-200e-496e-a183-248291a299de">

**Comment by tinohager** [comment]
The WebApi currently shows the day when the holiday is actually celebrated, with the shift.

**Comment by a-nasstrom** [comment]
this is confusing. maybe possible to add a separate field that will be responsible for the day of celebration?

**Comment by tinohager** [comment]
Yes, since version 2 of Nager.Date. Do we have this information available for the. It is planned to make this information available via the RestAPI as well.
